pkg/api/v1/testkube/model_test_step_execution_request.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Testkube API
+ *
+ * Testkube provides a Kubernetes-native framework for test definition, execution and results
+ *
+ * API version: 1.0.0
+ * Contact: testkube@kubeshop.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package testkube
+
+// test step execution request body
+type TestStepExecutionRequest struct {
+	// test execution custom name
+	Name string `json:"name,omitempty"`
```

Comment:
should be regenerated

Reply:
really good catch, the type was renamed and the generator did not delete the file